docs/guides/getting-started/prerequisites.md

```diff
@@ -3,22 +3,204 @@ sidebar_position: 1
+
+```text
+Rust is installed now. Great!
+```
+
+You also need to install a couple of system dependencies, such as a C compiler and `webkit2gtk`. Below are commands for a few popular distributions.
+
+<Tabs>
+  <TabItem value="debian" label="Debian" default>
+
+```console
+sudo apt update
+sudo apt install libwebkit2gtk-4.0-dev \
+    build-essential \
+    curl \
+    wget \
+    libssl-dev \
+    libgtk-3-dev \
+    libappindicator3-dev \
+    librsvg2-dev
+```
+
+  </TabItem>
+  <TabItem value="arch" label="Arch">
+
+```console
+sudo pacman -Syu
+sudo pacman -S --needed \
+    webkit2gtk \
+    base-devel \
+    curl \
+    wget \
+    openssl \
+    appmenu-gtk-module \
+    gtk3 \
+    libappindicator-gtk3 \
+    librsvg \
+    libvips
+```
+
+  </TabItem>
+  <TabItem value="fedora" label="Fedora">
+
+```console
+sudo dnf check-update
+sudo dnf install webkit2gtk3-devel.x86_64 \
+    openssl-devel \
+    curl \
+    wget \
+    libappindicator-gtk3 \
+    librsvg2-devel
+sudo dnf group install "C Development Tools and Libraries"
+```
+
+  </TabItem>
+</Tabs>
+
+### Windows Subsystem for Linux (WSL)
+
+To run a graphical application with WSL, you need to download one of these X servers: Xming, Cygwin X, and vcXsrv. Since vcXsrv has been used internally, it's the one we recommend installing.
+
+#### WSL Version 1​
+
+Open the X server and then run the following in the terminal:
+
+```console
+export DISPLAY=:0
+```
+
+You should now be able to run any graphical application via the terminal.
+
+#### WSL Version 2
+
+You'll need to run a command that is slightly more complex than WSL 1:
+
+```console
+export DISPLAY=$(cat /etc/resolv.conf | grep nameserver | awk '{print $2}'):0
+```
+
+You will also need to add `-ac` to the X server as an argument. If for some reason this command doesn't work you can use an alternative command such as:
+
+```console
+export DISPLAY=$(cat /etc/resolv.conf | grep nameserver | sed 's/.* //g'):0
+```
+
+or you can manually find the Address using:
+
+```console
+cat /etc/resolve.conf | grep nameserver
+```
+
+:::tip
+
+Don't forget that you'll have to use the `export` command anytime you want to use a graphical application for each newly opened terminal.
+
+You can download some examples to try with `sudo apt-get install x11-apps` (xeyes is a good choice). It can be handy when troubleshooting WSL issues.
+
+:::
+
+## Updating and Uninstalling
+
+Tauri and its components can be manually updated by editing the `Cargo.toml` file or running the `cargo upgrade` command that is part of the [`cargo-edit`] tool. Open a terminal and enter the following command:
+
+```console
+cargo upgrade
+```
+
+Updating Rust itself is easy via `rustup`. Open a terminal and run the following command:
+
+```console
+rustup update
+```
+
+`rustup` can also be used to uninstall Rust from your machine fully:
+
+```console
+rustup self uninstall
+```
+
+<!-- TODO: Unistall Webview2 -->
```

Comment:
Should we open an issue on the repo for this so we come back to it?

Reply:
Just remove that part lol. webview2 isn't reallyyy supposed to be uninstalled anyway (and you can't even do that on win11). Also it's a "normal" uninstallation process every windows user should know.